docs/tutorials/mongoid-documents.txt

```diff
@@ -415,6 +415,33 @@ For cases when you do not want to have ``BSON::ObjectId`` ids, you can override
      field :_id, type: String, default: ->{ name }
    end
 
+A Note about the BSON Symbol type
+*********************************
+
+Because the BSON specification deprecated the BSON symbol type, the `bson` gem will serialize Ruby
+symbols into BSON strings when used on its own. However, in order to maintain backwards
+compatibility with older datasets, `mongo` gem overrides this behavior to serialize Ruby symbols as
+BSON symbols. This is necessary to be able to specify queries for documents which contain BSON
+symbols as fields.
```

Comment:
Maybe add a sentence here saying that although Mongoid still allows you to define models with Symbol field types, this could present problems when using other mongodb tools that have actually removed support for the type.